server/patches/server/level/ServerPlayer.patch

```diff
@@ -1,14 +1,32 @@
 --- a/net/minecraft/server/level/ServerPlayer.java
 +++ b/net/minecraft/server/level/ServerPlayer.java
-@@ -137,6 +137,7 @@
+@@ -137,6 +137,8 @@
  
  public class ServerPlayer extends Player implements ContainerListener {
  
 +    public java.util.OptionalInt viewDistance = java.util.OptionalInt.empty(); // Loom
++    public Component tabListName = null; // Loom
```

Comment:
This should be in the PlayerImpl class. We want to make sure we keep diffs as small as possible, and when possible, things like this should be in the Loom implementation.